sopel/modules/clock.py

```diff
@@ -299,20 +309,24 @@ def update_channel_format(bot, trigger):
         bot.db.set_channel_value(trigger.sender, 'time_format', old_format)
         return
     bot.db.set_channel_value(trigger.sender, 'time_format', tformat)
+
+    help_prefix = bot.settings.core.help_prefix
+    set_command = '%ssettz' % help_prefix
+    channel_command = '%schanneltz' % help_prefix
     bot.reply("Got it. Times in this channel  will now appear as %s "
               "unless a user has their own format set. (If the timezone"
-              " is wrong, you might try the settz and channeltz "
-              "commands)" % timef)
+              " is wrong, you might try the %s and %s "
+              "commands)" % (timef, set_command, channel_command))
 
 
 @module.commands('getchanneltimeformat', 'getctf')
-@module.example('.getctf [channel]')
+@module.example('.getctf #sopel', user_help=True)
+@module.example('.getctf', user_help=True)
 def get_channel_format(bot, trigger):
-    """
-    Gets the channel's preferred time format; will return current channel's if
-    no channel name is given.
-    """
+    """Gets the channel's preferred time format
 
+    It returns current channel's if no channel name is given.
```

Comment:
```suggestion
    It returns the current channel's if no channel name is given.
```

Did I miss this earlier? Damn.